homeassistant/components/dialogflow/__init__.py

```diff
@@ -153,16 +169,43 @@ def __init__(self, parameters, api_version):
     def add_speech(self, text):
         """Add speech to the response."""
         assert self.speech is None
-
         if isinstance(text, template.Template):
             text = text.async_render(self.parameters, parse_result=False)
-
         self.speech = text
+        
+    def add_next_scene(self, next_scene):
+        """Add next_scene to the response."""
+        self.next_scene = next_scene
 
     def as_dict(self):
         """Return response in a Dialogflow valid dictionary."""
+        returnValue = None
         if self.api_version is V1:
-            return {"speech": self.speech, "displayText": self.speech, "source": SOURCE}
+            returnValue = {"speech": self.speech, "displayText": self.speech, "source": SOURCE}
```

Comment:
Actually, this style of variable naming is not the praxis in home assistant. Either male it return_value or just result.

Reply:
Also, we prefer early return value and we can't be logging warnings for normal behaviour.

Reply:
change returnValue by return_value